conans/test/unittests/client/conf/config_installer/test_install_folder.py

```diff
@@ -0,0 +1,22 @@
+# coding=utf-8
+
+import unittest
+import os
+from os import listdir
+from os.path import isfile, join
+
+from conans.client.conf.config_installer import tmp_config_install_folder
+from conans.test.utils.tools import TestClient
+
+
+class InstallFolderTests(unittest.TestCase):
+
+    def test_unique_install_folder(self):
+        client = TestClient()
+
+        with tmp_config_install_folder(client.cache) as tmp_folder_first:
+            open(os.path.join(tmp_folder_first, "foobar.txt"), "w+")
+            with tmp_config_install_folder(client.cache) as tmp_folder_second:
```

Comment:
Check that there are no files after the second _opening_, the `tmp_folder_second` must be empty

Reply:
good idea, I'll do

Reply:
done